ui/src/app/views/workspace-edit/component.html

```diff
@@ -102,3 +102,11 @@
     <p *ngIf="this.valueNotEntered;">* is a required field</p>
   </div>
 </div>
+<clr-modal [(clrModalOpen)]="workspaceCreationError">
+  <h3 class="modal-title">Error:</h3>
+  <div class="modal-body">Could not create workspace.</div>
+  <div class="modal-footer">
+    <button type="button" class="btn btn-primary" (click)="navigateBack()">Cancel Creation</button>
+    <button type="button" class="btn btn-primary" (click)="resetWorkspaceCreation()">Try Again</button>
```

Comment:
What do you think of "Keep Editing"? I would expect "Try Again" to do basically what `retryApi` does -- just keep issuing RPCs without my intervention.

Reply:
I agree, I think "Keep Editing" is better. Making the change.